apps/ledger-live-mobile/src/screens/SendFunds/02-SelectRecipient.tsx

```diff
@@ -341,7 +341,12 @@ export default function SendSelectRecipient({ navigation, route }: Props) {
               testID="recipient-continue-button"
               type="primary"
               title={<Trans i18nKey="common.continue" />}
-              disabled={debouncedBridgePending || !!status.errors.recipient || !!memoTag?.error}
+              disabled={
```

Comment:
just for readability, we can extract this above :)

Reply:
Done!